### PR TITLE
ceph_objectstore_tool.py: Don't use btrfs on FreeBSD

### PR DESCRIPTION
**Pull request by wjwithagen**

### CODE REVIEW

File: src/test/ceph_objectstore_tool.py [hunk 1]
```diff
@@ -1909,6 +1909,7 @@ def remove_btrfs_subvolumes(path):
         status = main(sys.argv[1:])
     finally:
         kill_daemons()
-        remove_btrfs_subvolumes(CEPH_DIR)
+        if platform.system() != "FreeBSD":
```
Comment (tchaikov) [inline src/test/ceph_objectstore_tool.py]: nit, could you move this check into `remove_btrfs_subvolumes()`?

**Comment by wjwithagen** [comment]
@tchaikov 
Ping?

**Comment by tchaikov** [comment]
lgtm @dzafman what do you think?

File: src/test/ceph_objectstore_tool.py [hunk 2]
```diff
@@ -1889,6 +1889,8 @@ def main(argv):
 
 
 def remove_btrfs_subvolumes(path):
+    if platform.system() != "FreeBSD":
```
Comment (dzafman) [inline src/test/ceph_objectstore_tool.py]: This shouldn't be: if platform.system() == "FreeBSD": ?

**Comment by dzafman** [comment]
@tchaikov I don't understand this change.  This looks like the purpose of the remove_btrfs_subvolumes() is exclusively for FreeBSD.  See my comment on the code.

**Comment by tchaikov** [comment]
@dzafman sorry, i should have checked this change more closely. yes, it should be 
```python
if platform.system() == "FreeBSD"
```

@wjwithagen could you fix this?

**Comment by wjwithagen** [comment]
@tchaikov @dzafman 
Right, Stupid copy and wrong paste, will fix.

**Comment by wjwithagen** [comment]
please retest this

**Comment by wjwithagen** [comment]
@tchaikov 
Ping?

**Comment by tchaikov** [comment]
retest this please.

**Comment by yuriw** [comment]
test this please

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2016-11-16_11:34:58-rados-wip-kefu-testing---basic-mira/

**Pull request merged**